### ISSUE TITLE
Remove warning screen, or reduce its use

### ISSUE THREAD

**Issue by zephyrr** [adiwg/mdEditor#144]
### Issue Description:

(Recording Deanne's request)

Whenever one clears cache, mdEditor reloads and pops up the warning screen.  This gets tiresome as I clear every time I switch projects.  (It says the non-warning version will be coming out last September).

You could simply remove it, but if it or some replacement is still important, could you make it show only once, at the start of a session?  Or upon version change (eg: store version in cache and display only if it has changed).

**Comment by jlblcc** [comment]
Display of the initial pop is determined by properties stored in the settings. It will show up if the mdEditor has been updated since the last access time(version is stored in settings). So if you clear the cache(and the settings), it will show on next load.

Maybe the best solution would be to add a "keep settings" option to the clear cache tool. Would that be a workable solution?

**Comment by zephyrr** [comment]
I like that; Deanne can also comment.

**Comment by deanned** [comment]
that sounds great!

**Comment by hmaier-fws** [comment]
Duplicate of #257. Closed by PR #385
